pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -518,7 +519,7 @@ class Build {
         context.node('master') {
             context.stage("installer") {
                 switch (buildConfig.TARGET_OS) {
-                    case "mac": buildMacInstaller(versionData); break
+                    case "mac": context.sh 'rm -f workspace/target/*.pkg; done'; buildMacInstaller(versionData); break
```

Comment:
Shouldn't this be in a cleanup function of somesort?  I'm curious as to why we don't explicitly do this for say, Windows.

Reply:
I'm honestly uncertain as to why we don't need to do this for Windows. The msi files don't seem to accumulate like the pkg files do.

I don't think this needs an entire function. I think the function should be consistent in how much trash it leaves behind, and since we don't create trash from a Linux run, we shouldn't do for a Mac run either.

That said, I see no reason we couldn't have a "cleanWs" at the start/end of a run. Perhaps an enhancement?

Reply:
Yeah we can open a separate issue for that I guess.